src/gl/specs/gl-api.spec

```diff
@@ -227,6 +227,12 @@ spec GL.API is
                             Index_Type : Unsigned_Numeric_Type;
                             Indices    : Low_Level.IntPtr) with
      Static => "glDrawElements", Wrapper => "GL.Objects.Buffers.Draw_Elements";
+   procedure Draw_Elements_Instanced (Mode       : Connection_Mode;
+                                      Count      : Size;
+                                      Index_Type : Unsigned_Numeric_Type;
+                                      Indices    : Low_Level.IntPtr;
+                                      Instances  : Size) with
+    Static => "glDrawElementsInstanced", Wrapper => "GL.Objects.Buffers.Draw_Elements_Instanced";
```

Comment:
This needs to be `Dynamic`, not `Static`.

Reply:
Fixed